pygmt/tests/test_coast.py

```diff
@@ -72,3 +73,35 @@ def test_coast_dcw_list():
         dcw=["ES+gbisque+pgreen", "IT+gcyan+pblue"],
     )
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_coast_clip_land():
+    """
+    Test to clip dry areas.
+    """
+    region = [-28, -10, 62, 68]
+    grid = load_earth_relief(resolution="10m", region=region)
+
+    fig = Figure()
+    fig.basemap(region=region, projection="M12c", frame=True)
+    fig.coast(resolution="l", clip="land")
+    fig.grdimage(grid=grid, shading=True, cmap="batlow")
```

Comment:
Setting `shading=True` is unnecessary and makes the test slower, so remove it.
```suggestion
    fig.grdimage(grid=grid, cmap="batlow")
```